kolibri/plugins/learn/assets/src/views/LibraryAndChannelBrowserMainContent.vue

```diff
@@ -32,7 +32,7 @@
     </CardGrid>
     <!-- large displays, list view -->
     <HybridLearningContentCardListView
-      v-for="content in contents"
+      v-for="(content, idx) in contents"
```

Comment:
This file does rather seem like it's acting as a responsive Card grid with a loading state? Could we do a rename to something that more resembles that?